ts/client/services/rpc.ts

```diff
@@ -20,12 +22,70 @@ export class RPC {
         fastify: FastifyInstance,
         transferPool?: ITransferPool
     ) {
+        fastify.register(cors, {
+            origin: (origin, cb) => {
+                if (/localhost/.test(origin)) {
+                    //  Request from localhost will pass
+                    cb(null, true);
+                    return;
+                }
+                // Generate an error on other origins, disabling access
+                cb(new Error("Not allowed"), false);
+            }
```

Comment:
This will fail for wallets/web app calls to testnet node deploys. I would just set this as a wildcard (`*`) origin. Later on we can make it configurable.

```suggestion
            origin: "*"
```